packages/fast-data-utilities-react/src/index.ts

```diff
@@ -0,0 +1,457 @@
+import * as React from "react";
+import { cloneDeep, get, isPlainObject, set } from "lodash-es";
+import * as tv4 from "tv4";
+
+export interface ChildOptionItem {
+    /**
+     * The name of the component
+     */
+    name?: string;
+
+    /**
+     * The React component
+     */
+    component: React.ComponentClass;
```

Comment:
you should change `React.ComponentClass` to `React.ComponentType` so that functional components are supported. Also I think that is a generic... is it not yelling at you?